src/framework/components/element/constants.js

```diff
@@ -20,22 +20,24 @@ export const ELEMENTTYPE_IMAGE = 'image';
 export const ELEMENTTYPE_TEXT = 'text';
 
 /**
- * Disable Aspect Ratio content resizing.
+ * Fit the content to the exact Element's bounding box.
  *
  * @type {string}
  */
-export const ELEMENT_IMAGE_ASPECT_NONE = 'none';
+export const ELEMENT_IMAGE_FIT_STRETCH = 'stretch';
```

Comment:
What about `FITMODE_STRETCH`?

Reply:
Yea, I like it! I was thinking of going with the `<component>_<subcomponent>_<property>_<enum value>` style of constants, but it can get quite verbose, and there's quite a few of these less-verbose types of enums. Will change.